migrator/README.md

```diff
@@ -0,0 +1,63 @@
+# UnifiedPush Server - Migration guide
+
+For the database migration you may need to run a simple shell script
+
+# 1.0.x users
+
+For the migration to our 1.0.3 release we are only supporting the 1.0.x series of the UnifiedPush Server:
+
+* 1.0.0
+* 1.0.1
+* 1.0.2
+
+In terms of supported databases, we are supporting MySQL 5.5 and PostgreSQL 9. 
+
+## Getting started
+
+Extract the `unifiedpush-migrator-dist.zip` file to your desired location. 
+
+_NOTE:_ It is recommended to shutdown the application server, while performing the migration.
+
+### Database Migration
+
+Below you find information on how to perform the migration on each of the supported databases.
+
+#### PostgreSQL
+
+Inside of the extracted zip file, there is a `liquibase-postgresql-example.properties`, copy that to `liquibase.properties`.
+
+    cp liquibase-postgresql-example.properties liquibase.properties
+
+Once done, you need to edit the new file to match your database credentials.
```

Comment:
we could add "and database name" , I had to change that since it was postfix with some unwanted extra chars